api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/operations/ContainerLoader.java

```diff
@@ -0,0 +1,117 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.components.operations;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.solr.common.StringUtils;
+
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Parameter;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.Data;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.DataStore;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.JsonContainerView;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.SimpleDataView;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.conversion.InitializationException;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.implementation.JsonContainer;
+import edu.cornell.mannlib.vitro.webapp.utils.configuration.Property;
+
+public class ContainerLoader extends AbstractOperation {
```

Comment:
For now it is only an operation component to load data into json that can be used in procedure execution logic.
In future it might also be extended to take specific json path to put data in json that can be used later by other components. Will see if we find something more effective for  Sparql, Solr and http client json's modification.